pages/events/_id/edit.vue

```diff
@@ -238,8 +238,15 @@ export default {
           },
           {
             name: 'price',
-            placeholder: 'Examples: 10EUR, 5USD, FREE',
-            description: 'Examples: 10EUR, 5USD, FREE',
+            type: 'price',
+            placeholder: 'Examples: 10, 5, 0',
+            description: 'Examples: 10, 5, 0',
+            options: [
+              { name: 'EUR' },
```

Comment:
use `label` and `value` fields, cuz now `EUR` is translated into ` E U R `

Reply:
i think there are some libraries available in npm where all currencies are listed, let's integrate one of them?

Reply:
I searched but I didn't like most of libraries . I didn't want just a list of currency list.  I thought adding symbol puts extra niceness to UI. I found a list and added it to under _use/currencies.js_ .

Reply:
how about using this package?
npm install vue-currency-input

Reply:
thanks for the advice. I couldn't come across them before. vue-currency-input seems nice actually and well maintained but vue-currency is very old and not maintained. maybe I can use vue-currency-input. @razbakov should I use it?